admin-tools/lib/src/main/scala/com/gu/mediaservice/BatchIndexHandler.scala

```diff
@@ -39,8 +39,9 @@ class BatchIndexHandler(cfg: BatchIndexHandlerConfig) {
   private val GetIdsTimeout = new FiniteDuration(GetIdsTimoutInMins, TimeUnit.MINUTES)
   private val GlobalTimeout = new FiniteDuration(GlobalTimoutInMins, TimeUnit.MINUTES)
   private val ImagesProjectionTimeout = new FiniteDuration(ProjectionTimoutInMins, TimeUnit.MINUTES)
+  private val gridClient = new GridClient()
 
```

Comment:
without new
  private val gridClient = GridClient()
and pass maxIdleConnections